src/haxeLanguageServer/features/completion/CompletionFeature.hx

```diff
@@ -280,7 +280,22 @@ class CompletionFeature {
                         case _: field.name;
                     }
                 },
-                range: data.replaceRange
```

Comment:
I'm seeing that occasionally the replaceRange is empty during class field completion requests.  It seems that normally you could infer a null replaceRange means that you don't replace anything at all.  The neovim languageserver plugin doesn't seem to infer this however, and expects it always to be present.  I'm not sure if we fill in a dummy here, or ask them to handle null replaceRange fields.